<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>521</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Poppins</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family:&quot;Poppins&quot;,sans-serif;
font-size:26px;
font-weight:bold;
color:#222;
text-align:center;
text-transform:uppercase;
letter-spacing:2px;
padding:10px;
border-radius:8px;
background-color:#f0f0f0;
border:2px solid #0072ff;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff5500;&quot;&gt;pdf / docx / ppt generation&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QFrame" name="dropArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>741</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #fafafa;
border: 2px dashed #bbb;
border-radius: 8px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Drop file here&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowse">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background-color:#4CAF50;color:white;border:none;padding:10px 20px;border-radius:6px;font-size:20px;font-weight:bold;}QPushButton:hover{background-color:#45a049;}QPushButton:pressed{background-color:#3e8e41;}
</string>
    </property>
    <property name="text">
     <string>Browse files</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineFilePath">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>270</y>
     <width>581</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{background:#fafafa;border:1px solid #ccc;border-radius:4px;padding:6px;font-size:13px;}QLineEdit:focus{border:1px solid #4CAF50;outline:none;}</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboType">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>340</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{background:#fafafa;border:1px solid #ccc;border-radius:4px;padding:6px;font-size:13px;}QComboBox:focus{border:1px solid #4CAF50;outline:none;}QComboBox::drop-down{border:none;width:20px;}</string>
   </property>
   <item>
    <property name="text">
     <string>PDF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DOCX</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PPT</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; color:#000000;&quot;&gt;File chosen :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Type to generate :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerate">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>330</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color:#2196F3;color:white;border:none;padding:10px 20px;border-radius:6px;font-size:14px;font-weight:bold;}QPushButton:hover{background-color:#1976D2;}QPushButton:pressed{background-color:#1565C0;}</string>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
